src/Datadog.Trace.ClrProfiler.Managed/Integrations/AspNet/AspNetWebApi2Integration.cs

```diff
@@ -225,7 +229,7 @@ private static Scope CreateScope(object controllerContext)
             return scope;
         }
 
-        private static void UpdateSpan(dynamic controllerContext, Span span)
+        private static void UpdateSpan(dynamic controllerContext, Span span, IEnumerable<KeyValuePair<string, string>> headerTags)
```

Comment:
I thought we got rid of dynamics?

Reply:
This is the last integration still using it, and @zacharycmontoya started a PR (#761) which removes it.